docs/sequenceDiagrams/seq-cep-10.4-action-flow.plantuml

```diff
@@ -0,0 +1,95 @@
+@startuml
+' declate title
+title 10.4. Notification Action Flow
+autonumber
+
+' declare actors
+collections "Notification-Topic" as topicNotify
+boundary "Central Event Processor" as CEP
+control "Action Agent" as AA
+database "CEP Store" as DB
+
+box "Central Services" #LightYellow
+participant topicNotify
+end box
+box "Central Event Processing Service" #LightGreen
+participant CEP
+participant AA
+participant DB
+end box
+
+' start flow
+Group Action Flow
+    activate CEP
+    CEP -> AA: Recieves an action event
```

Comment:
Just "Action event" or "Sends an action event" (but send is implied by the arrow)

Reply:
Updated to
CEP -> AA: Action event